gitindex/clone.go

```diff
@@ -38,7 +59,7 @@ func CloneRepo(destDir, name, cloneURL string, settings map[string]string) (stri
 
 	repoDest := filepath.Join(parent, filepath.Base(name)+".git")
 	if _, err := os.Lstat(repoDest); err == nil {
-		return "", nil
+		return "", updateZoektGitConfig(repoDest, settings)
```

Comment:
this sort of flow makes it a bit tricky to read / easy to miss we do this.
```suggestion
		// Repository exists, ensure settings are in sync
		if err := updateZoektGitConfig(repoDest, settings); err != nil {
			return "", fmt.Errorf("failed to update repository settings: %w", err)
        }
		return "", nil
```

Reply:
Nice, more future-readable. Thanks!

Reply:
I pushed a change which added the `fmt` package and ran `gofmt` on the file.